python/sparkdl/graph/graph_factory.py

```diff
@@ -0,0 +1,136 @@
+#
```

Comment:
still woud like a better name for the module since "factory" sounds like https://en.wikipedia.org/wiki/Factory_method_pattern